plugins/vesta-change-password/index.php

```diff
@@ -42,7 +42,7 @@ public function configMapping()
         return array(
             \RainLoop\Plugins\Property::NewInstance('vesta_host')->SetLabel('Vesta Host')
                 ->SetDefaultValue('')
-                ->SetDescription('Allowed patterns: {user:host-imap}, {user:host-smtp}, {user:domain}'),
+                ->SetDescription('Ex: localhost or domain.com'),
```

Comment:
But these patterns still allowed.

Reply:
The connection should only be a hostname, and while those examples of host values are correct, if the user enters username:hostname as their value, like described above, the connection will fail.